Octokit/Models/Response/OrganizationSecret.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Octokit.Internal;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Represents an organization secret.
+    /// Does not contain the secret value
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class OrganizationSecret : RepositorySecret
+    {
+        public OrganizationSecret() { }
+
+        public OrganizationSecret(string name, DateTime createdAt, DateTime updatedAt, string visibility, string selectedRepositoriesUrl)
+        {
+            Name = name;
+            CreatedAt = createdAt;
+            UpdatedAt = updatedAt;
+            Visibility = visibility;
+            SelectedRepositoriesUrl = selectedRepositoriesUrl;
+        }
+
+        /// <summary>
+        /// The visibility level of the secret within the organization
+        /// </summary>
+        public string Visibility { get; protected set; }
```

Comment:
We’ve recently changed to private setters for response models. 👍

Reply:
I've changed to private setters in all introduced models! Thanks.

Reply:
Okay, not quite _all_ introduced response models. `OrganizationSecret` inherits from `RepositorySecret` so `RepositorySecret` needs protected setters.

Reply:
👍